streampark-console/streampark-console-service/pom.xml

```diff
@@ -383,6 +383,12 @@
             <version>${swagger.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.casbin</groupId>
+            <artifactId>casdoor-spring-boot-starter</artifactId>
+            <version>1.5.1</version>
```

Comment:
Please extract version to <properties/> frame

Reply:
ok